core/src/test/kotlin/com/linecorp/kotlinjdsl/querydsl/function/QueryDslImplFunctionTest.kt

```diff
@@ -0,0 +1,54 @@
+package com.linecorp.kotlinjdsl.querydsl.function
+
+import com.linecorp.kotlinjdsl.query.clause.select.SingleSelectClause
+import com.linecorp.kotlinjdsl.query.spec.expression.ColumnSpec
+import com.linecorp.kotlinjdsl.query.spec.expression.EntitySpec
+import com.linecorp.kotlinjdsl.query.spec.expression.FunctionSpec
+import com.linecorp.kotlinjdsl.query.spec.expression.LiteralSpec
+import com.linecorp.kotlinjdsl.querydsl.QueryDslImpl
+import com.linecorp.kotlinjdsl.querydsl.expression.column
+import com.linecorp.kotlinjdsl.test.WithKotlinJdslAssertions
+import org.junit.jupiter.api.Test
+
+internal class QueryDslImplFunctionTest : WithKotlinJdslAssertions {
```

Comment:
-- korean
Expression 관련 테스트라서 `QueryDslImplExpressionTest` 클래스에 작성해도 좋을 것 같습니다.
`QueryDslImplExpressionTest` 클래스가 너무 커서 따로 빼주신 건가요?

-- english
It's an expression-related test, so it can be moved to `QueryDslImplExpressionTest`.
Is it separated because the `QueryDslImplExpressionTest` is too big?

Reply:
-- korean
QueryDslImplExpressionTest 로 테스트를 이동했습니다.

-- english
move test to QueryDslImplExpressionTest

Reply:
-- korean
머지하면서 삭제 되었던 테스트가 살아난 것 같습니다.

-- english
The deleted test have came back due to merge.

Reply:
-- korean
반영했습니다
-- english
applied